docs/user/security-log.rst

```diff
@@ -1,16 +1,19 @@
 Security Log
 ============
```

Comment:
I remember you we could change the title to something like: "Security log: audit organization's account".

Reply:
Good idea to promote "auditing" to the title - how about "Security log and auditing"?

Reply:
Yea, I tried that, but I was worried it didn't match the other titles in our docs.